### ISSUE TITLE
Copter: add missing motor diagrams

### ISSUE THREAD

**Issue by rmackay9** [ArduPilot/ardupilot_wiki#1875]
We are missing the following motor diagrams:

- Hexacopter H.  This is the same as Hexacopter X except motors 1 and 2 are closer to the vehicle's center
- BiCopter

**Comment by rmackay9** [comment]
@lvale told me today that there are others missing include DJI motor orders

**Comment by auturgy** [comment]
Yes, and betaflight motor order

Regards,

James

**Comment by Hwurzburg** [comment]
add reverse tricopter

**Comment by IamPete1** [comment]
to expand, Motor 1 and 2 remain on the right and left respectively as with standard tricopter. the tail motor 4 is now at the front https://github.com/ArduPilot/ardupilot/pull/10846

**Comment by IamPete1** [comment]
not copter but we should also add diagrams for the rover omni frames

**Comment by rmackay9** [comment]
@IamPete1, we have at least some of the rover omni frame here I think: http://ardupilot.org/rover/docs/rover-motor-and-servo-connections.html#omni-vehicles

**Comment by IamPete1** [comment]
I was looking at the second page, so missed them completely, in any-case were missing the 4 inline mecanum wheels that I think we support?

**Comment by rmackay9** [comment]
@IamPete1, no, I think that's one of the existing omni vehicles... maybe the OmniX.  I'm not totally sure but I don't remember us having to make any code changes for the person that got the omni rover working.

**Comment by rmackay9** [comment]
We are also missing these for Quad:

- BetaFlight X
- DJI X
- CW X

**Comment by Hwurzburg** [comment]
@rmackay9  if you can hand sketch, and send me a photo, I will work on these....could save us some $$ right?

**Comment by rmackay9** [comment]
FYI the DJIX configuration came up today in this support thread: https://discuss.ardupilot.org/t/copter-4-0-1-released/51505/19

**Comment by rmackay9** [comment]
This is the link to where we store the configurations: https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Motors/AP_MotorsMatrix.cpp#L498

**Comment by Hwurzburg** [comment]
Closing as addressed. Checked `copter/source/docs/connect-escs-and-motors.rst`: all items from the checklist are present — Hexa H, BiCopter, Quad BetaFlight X, Quad DJI X, Quad CW X, Hexa CW X, Octa CW X, OctaQuad CW X, and Quad BF Reversed. The follow-up asks in this thread are also covered: DJI motor order (Quad/Hexa/Octo X (DJI)), BetaFlight order, and reverse tricopter (Tricopter Pitch Reversed).

The Rover omni-frame diagram gap raised by @IamPete1 in this thread is real and still open — split out to #7913 since it's a distinct, Rover-specific ask.
